mandatory/2-function-creation.js

```diff
@@ -5,7 +5,14 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  for (let i = 0; i < strArr.length;i++) {
+    strArr[i] = strArr[i].trim();
+    strArr[i] = strArr[i].replace(`/`, ``);
+    strArr[i] = strArr[i].toLowerCase();
```

Comment:
good work. you can apply these functions in one go .trim().replace().toLowerCase()